src/config.yml

```diff
@@ -213,14 +213,20 @@ announcer:
     - 'Be sure to visit our forums at &6http://totalfreedom.boards.net/'
     - 'You can always review the server rules here: &6http://totalfreedom.me/'
     - 'If you are not OP, be sure to ask!'
+    - 'Players with their prefix/tag as '[Helper]' aren't staff and it isn't a rank, you can set your own tag by doing /tag set <tag>'
     - 'Somebody breaking the rules?  Report it!  /report <user> <reason>'
     - 'Griefing is not allowed!'
+    - 'Spamming is not allowed!'
+    - '&4TotalFreedom &b- Giving players joy since 2010!'
+    - '"Logic is just an illusion, don't expect to benefit from logic, because it doesn't exist in this world"'
+    - '"Man lives by imagination"'
     - 'Hacking is allowed!'
     - 'Interested in becoming admin? Do "/ai" for more information!'
     - 'You may view all online administrators via "/list -a"'
     - 'Save your buildings via WorldEdit!  http://totalfreedom.me for more information!'
     - 'Famous players, such as Notch, are always fake!  We are an offline/cracked server!'
     - 'You may contact TotalFreedoms Support on Twitter!  pravi.us/nGS'
+    - 'You may only advertise once, continuous advertisements will result in a mute or tempban!'
     - 'You may download TotalFreedomMod here: pravi.us/V8c'
```

Comment:
Maybe update the pravius link to go to https://github.com/TotalFreedom/TotalFreedomMod/releases/latest instead of https://github.com/TotalFreedom/TotalFreedomMod/releases/ just an idea :)